src/web/run.rs

```diff
@@ -42,7 +47,9 @@ pub(crate) fn run_web(
     let web_bundle = build_web(&mut build_args, metadata, bin_target)?;
 
     let port = web_args.port;
-    let url = format!("http://localhost:{port}");
+    let host = IpAddr::from_str(&web_args.host).expect("Failed to parse host address");
```

Comment:
```suggestion
    let host = IpAddr::from_str(&web_args.host).context("failed to parse host address")?;
```

We try to avoid panicking if possible! This switches it to return an `Err` using [`anyhow`](https://docs.rs/anyhow/latest/anyhow/) :)